lib/modules/dosomething/dosomething_signup/dosomething_signup.module

```diff
@@ -518,13 +524,22 @@ function dosomething_signup_entity_insert($entity, $type) {
   }
 
   // Send relevant third-party subscribe requests.
-  dosomething_signup_third_party_subscribe($account, $node);
+  dosomething_signup_third_party_subscribe($account, $node, $transactionals, $source);
 }
 
 /**
  * Sends third-party subscription requests for given $account and $node.
+ *
+ * @param object $account
+ *    User details who signed up for a campaign.
+ * @param object $node
+ *   Details about the campaign the user signed up to.
+ * @parm bool $trasactionals
+ *    A flag to prevent sending transactionals when FALSE. Defaults to TRUE when NULL.
+ * @parm string $source
+ *    A value defining where the request to sign a user up to a campaign originated.
  */
-function dosomething_signup_third_party_subscribe($account, $node) {
+function dosomething_signup_third_party_subscribe($account, $node, $transactionals = NULL, $source = NULL) {
```

Comment:
Let's keep `$source` argument before `$transactionals`, as it is originally in `dosomething_signup_create()`;